flashloan-detector/src/agent.js

```diff
@@ -83,16 +86,22 @@ function provideHandleTransaction(helper, getFlashloans) {
     }
 
     if (!totalNativeProfit.isZero()) {
-      nativeUsdProfit = await helper.calculateNativeUsdProfit(totalNativeProfit, chain);
+      nativeUsdProfit = await helper.calculateNativeUsdProfit(totalNativeProfit, nativeToken);
     }
 
     const totalProfit = tokensUsdProfit + nativeUsdProfit;
     const percentage = (totalProfit / totalBorrowed) * 100;
 
-    if (percentage > PERCENTAGE_THRESHOLD && totalProfit > PROFIT_THRESHOLD) {
+    console.log('Chain     :', chain);
+    console.log('TX hash   :', txEvent.hash);
+    console.log('Borrowed  :', totalBorrowed.toFixed(2));
+    console.log('Profit    :', totalProfit.toFixed(2));
+    console.log('Percentage:', percentage.toFixed(2));
```

Comment:
Is this for logging purposes?

Reply:
Yes. In case there is a flashloan that isn't flagged in the future